### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
- Store chunkhash in query string (just like inertiajs/inertia-vue#41)
- Add docs about the Mix 'babelConfig' option (just like inertiajs/inertia-vue@3c74a86)
- Change `.jsx` extension  to `.js`

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -111,12 +119,12 @@ import { render } from 'react-dom'
 
 const app = document.getElementById('app')
 
-const files = require.context('./', true, /\.jsx$/i)
+const files = require.context('./', true, /\.js$/i)
```
Comment (reinink) [inline readme.md]: I was wondering if it had to be `.jsx`.

Comment (MichaelDeBoey) [reply readme.md]: It makes it clearer that the file is exporting a jsx component, but what I've seen is that it's more common to just use `.js` in the React ecosystem.

Comment (sebastiandedeyne) [reply readme.md]: Both are allowed, but `js` is more common these days :)

Comment (reinink) [reply readme.md]: Dig it.

**Comment by sebastiandedeyne** [comment]
Thanks!

**Pull request merged**